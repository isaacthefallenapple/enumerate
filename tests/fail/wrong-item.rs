use enumerate::Enumerate;

fn main() {
    #[derive(Enumerate)]
    struct S {
        a: (),
        b: (),
        c: (),
    }
}
