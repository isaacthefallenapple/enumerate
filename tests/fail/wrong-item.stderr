error: Expected `enum` found `struct`.
 --> $DIR/wrong-item.rs:5:5
  |
5 |     struct S {
  |     ^^^^^^
